src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private final int base;
+    private final int height;
+
+    public IsoscelesTrapezoid(String color, int base) {
+        super(color);
+        this.base = base;
+        this.height = (int) ((base * Math.sqrt(3) / 2));
+    }
+
+    @Override
+    public double area() {
+        return 0.5 * base * height;
```

Comment:
The area calculation seems to be incorrect for an isosceles trapezoid. It should take into account both bases and the height. Please revise the formula.